cmd/imports.go

```diff
@@ -42,4 +42,5 @@ import (
 	_ "github.com/Azure/azure-storage-fuse/v2/component/file_cache"
 	_ "github.com/Azure/azure-storage-fuse/v2/component/libfuse"
 	_ "github.com/Azure/azure-storage-fuse/v2/component/loopback"
+	_ "github.com/Azure/azure-storage-fuse/v2/component/xload"
```

Comment:
Based on a cli, the component shall be auto inserted in the pipeline. Check mount.go for block-cache addition. If user give "--preload" parameter then this mode shall get enabled and we shall auto create pipeline for prefetching.

Reply:
Added "--preload" flag to add xload to pipeline